gluten-data/src/main/java/io/glutenproject/columnarbatch/ColumnarBatches.java

```diff
@@ -218,10 +222,35 @@ private static ColumnarBatch offload(BufferAllocator allocator, ColumnarBatch in
     }
   }
 
+  public static Iterator<InternalRow> emptyRowIterator(ColumnarBatch batch) {
```

Comment:
@jinchengchenghh 

Should we use signature like

```
Iterator<InternalRow> emptyRowIterator(int maxRows)
```
? Which seems to be more explainable.

Reply:
Yes, next PR.

Reply:
It is only used in ColumnartoRow, if change input to maxRows, which file should I put it? Do you have some suggestions?